crates/state/src/resources/schema.sql

```diff
@@ -0,0 +1,31 @@
+BEGIN TRANSACTION;
+
+DROP TABLE IF EXISTS "commits";
+CREATE TABLE IF NOT EXISTS "commits" (
+	"id" INTEGER NOT NULL UNIQUE,
+	"signature" BLOB NOT NULL CHECK(length("signature") == 32),
+	PRIMARY KEY("id")
+);
+
+DROP TABLE IF EXISTS "values";
+CREATE TABLE IF NOT EXISTS "values" (
+	"id" INTEGER NOT NULL UNIQUE,
+	"commit_id"	INTEGER,
+	"key_hash" BLOB NOT NULL,
```

Comment:
It's a shame we can't tell SQLite that `key_hash` is of a fixed-size.

Reply:
I'll add a `check`, that's the best we can do.